src/styles/main.scss

```diff
@@ -12,3 +12,24 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+@media screen and (min-width: 768px) and (max-width: 1023px) {
```

Comment:
You don't need to set a gap at media-queries here like (min-width: 768px) and (max-width: 1023px)
You can just use as waterfall 
Like 
```
@media screen and (min-width: 768px) {
[...styles that rewrite previous ones]
} 
@media screen and (min-width: 1024) {
[...styles that rewrite previous ones]
}
```

Reply:
Use it below as well